lib/src/commands/sublist.dart

```diff
@@ -112,17 +141,20 @@ class SublistQueryProcessor extends QueryProcessor {
           }
         }
       } else {
-        // We don't care about updates. Paths don't really change.
-        if (holders.containsKey(path)) {
-          return;
-        }
+        SublistHolder holder = holders[path];
 
-        var holder = new SublistHolder(path);
-        holder.listSub = holder.create(this).listen((QueryUpdate update) {
-          controller.add(update);
-        });
+        if (holder == null) {
+          holder = new SublistHolder(path);
 
-        holders[path] = holder;
+          holder.listSub = holder.create(this).listen((QueryUpdate update) {
+            controller.add(update);
+          });
+
+          holder.controller = controller;
+          holders[path] = holder;
+        }
+        holder.oldUpdate = update;
+        holder.onOldUpdate();
```

Comment:
I'd personally pass the update to `onOldUpdate` and assign it in there. If it makes sense.

Reply:
I could do that, I'll change it.